enterprise/utils.py

```diff
@@ -1419,7 +1419,9 @@ def enroll_user(enterprise_customer, user, course_mode, *course_ids, **kwargs):
     succeeded = True
     for course_id in course_ids:
         try:
-            enrollment_client.enroll_user_in_course(user.username, course_id, course_mode)
+            enrollment_client.enroll_user_in_course(user.username, course_id, course_mode, cohort=None,
+                                                    enterprise_uuid=str(enterprise_customer_user.uuid)
+                                                    if hasattr(enterprise_customer_user, 'uuid') else None)
```

Comment:
[style nitpick] Same long-line breaking request as above.

[style nitpick] The `cohort` parameter defaults to `None`, so you don't have to include it in the call, here.

[Fix] I think you want
```python
enterprise_uuid=str(enterprise_customer_user.enterprise_customer.uuid)
```
here - the `EnterpriseCustomerUser` model doesn't have a `uuid` field, so you can drop the conditional.

Reply:
@iloveagent57  It's done